Eastern-Research-Group/mywaterway
# Feature/national data json

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3422228

## Main Changes:
* Moves NARS data on National page into a JSON file. The data is used to create the accordions on first tab of the National page.
* Simplifies the way the "Learn about the health of our waters" section is rendered. Accordions are now generated by looping over arrays of objects in the JSON file for more flexibility in the future. 

## Steps To Test:
1. Navigate to http://localhost:3000/national
2. Check that accordions are functioning as expected: clicking an accordion opens and collapses it and the Expand/Collapse All button works.
3. Check that the content of the section is correct against the live site.
4. Open the Network tab and search "proxy". Block the NARS.json service call. 
5. The NARS section should display a spinner until it stops retrying, then it should display an error box.
